Microsoft.Toolkit.Uwp.UI/ImageCache/ImageCache.cs

```diff
@@ -156,7 +176,7 @@ public static int MaxMemoryCacheSize
             catch (Exception ex)
             {
                 System.Diagnostics.Debug.WriteLine(ex.Message);
-                return null;
+                throw ex;
```

Comment:
I've marked the method internal. It is used in ImageEx to signal ImageFailedEx event.